lib/api-wmata.js

```diff
@@ -0,0 +1,24 @@
+var request = require('request');
+
+var props = require('../resources/properties');
+var secrets = require('../secret/secrets');
+
+var wmataReq = request.defaults({
+  baseUrl: props.wmata_base_url,
+  qs: secrets.apiKey
+});
+
+var WmataApi = function() {};
+
+WmataApi.prototype.get = function(endpoint, response, callback) {
+  wmataReq(endpoint, function(error, res, body) {
+    if (!error && res.statusCode === 200) {
+      callback(JSON.parse(body));
+    } else {
+      console.error(endpoint.concat(': Error with WMATA'), error);
+      response.tell(props.wmataErrorSpeechOutput);
+    }
+  });
+};
+
+module.exports = new WmataApi();
```

Comment:
Absolutely. I was just thinking that from an interface point of view, exporting the specific wmataGet method is crying for a future refactor if we found there were more methods needed related to WMATA API. This is a little premature refactor. My thought was that this has to be a best practice, but maybe it isn't.